### DISCUSSION TITLE
Nanite-like LOD system

### DISCUSSION THREAD

**Discussion by The-Ladle** [godotengine/godot-proposals#5842] [3D]
Sorry if this issue already exists. I don't know if devs have the capacity for a system like this, but this would be a cool addition.

**Comment by Zireael07** [comment]
Nanite would be an insanely huge undertaking, and the contributor team is already shortstaffed relative to the amount of bugs and feature requests made

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/2793.
